src/index.ts

```diff
@@ -103,7 +103,13 @@ function serveSinglePageApp(request: Request, options?: Partial<Options>): Reque
  * @param {Object | string} [options.ASSET_NAMESPACE] the binding to the namespace that script references
  * @param {any} [options.ASSET_MANIFEST] the map of the key to cache and store in KV
  * */
-const getAssetFromKV = async (event: FetchEvent, options?: Partial<Options>): Promise<Response> => {
+
+type Evt = {
+  request: Request
+  waitUntil: (promise: Promise<any>) => void
+}
```

Comment:
Why custom `Evt` type? You know there are [`@cloudflare/workers-types`](https://github.com/cloudflare/workers-types) out there, right?

Reply:
I chose the word "Evt" to be familiar, and `@cloudflare/workers-types` won't ever define the types for kv-asset-handler.